test/unit/app.spec.js

```diff
@@ -37,6 +37,26 @@ describe('App', function() {
     });
   });
 
+  describe('when restarting the app', function() {
```

Comment:
Ah, yeah that makes sense. Plus it has a lot of the stubbing work done there, no need for duplicating that.